recipe/meta.yaml

```diff
@@ -20,11 +20,14 @@ requirements:
   host:
     - pip
     - pytest-runner
-    - python  >=3.7
+    - python {{ python_min }}
```

Comment:
I've just noticed this changed from `>=3.7` to just `3.7`. Is this only needed for the Azure CI runner that conda-forge uses?

Reply:
Ok, it matches the example here so it must be ok https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

Reply:
I asked the folks on the Conda Forge channel:

> I suspect so that the package is built with the min version